spec/rails_helper.rb

```diff
@@ -5,6 +5,8 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'simplecov'
+SimpleCov.start 'rails'
```

Comment:
what about non-rails tests?

Reply:
I think it will work fine? Or not? we have same in our project